katello/katello-backup

```diff
@@ -21,7 +21,7 @@ optparse = OptionParser.new do |opts|
       opts.abort("Previous backup directory does not exist: #{dir_path}")
     end
   end
-  
+
   opts.on("--online-backup", "Backup Pulp data online") do |online|
```

Comment:
does this need to be altered to be something generic like "perform online backup"?